pkg/project/auth/cache_test.go

```diff
@@ -0,0 +1,143 @@
+package auth
+
+import (
```

Comment:
A few return it on List.  Look at fake-namespaces.go.  I know others do as well. 

Sent from my iPhone

> On Feb 20, 2015, at 7:53 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/project/auth/cache_test.go:
> 
> > @@ -0,0 +1,143 @@
> > +package auth
> > +
> > +import (
> I need the Fake client for PolicyBindings and Policy objects to actually support returning a Fake value
> 
> I wasn't planning it, but I'm not necessarily opposed. It doesn't look like any of the Fake clients in origin or in kubernetes do that. Do you have some some example of what you want?
> 
> —
> Reply to this email directly or view it on GitHub.
>